app/views/admin/users/_search_form_for.html.slim

```diff
@@ -0,0 +1,5 @@
+= search_form_for @search, url: admin_users_path do |f|
+  .field
```

Comment:
better use bootstrap
smth like this 
.col-lg-6
  form.input-group
    = search_form_for @search, url: admin_users_path do |f|
      = f.text_field :first_name_cont, :placeholder => "Search for...", class: "form-control"
      span.input-group-btn
        = f.submit "Go!", class: 'btn btn-default type="button"'